ajax/libs/cartodb.js/packages.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "cartodb.js",
```

```suggestion
  "name": "carto.js",
```

This library seems to go by carto.js now, not cartodb.js?

(_Directory this file is in will also need renaming_)